_gsocblogs/2022/blog_EtherpadPlugin_MohammadWarid.md

```diff
@@ -46,7 +46,33 @@ This implementation served well for the project which meant that the metadata ca
 
 Currently, I am investigating the WOPI server functionality for receiving the metadata and handling it, after which, I will move towards testing the complete workflow.
 
+# Post Midterm Period
+
+## Support for multi-user collaboration
+The main goal for the project was an in-app workflow, targeted to onboard new users over the etherpad each possessing a unique ID. This was achieved by fetching query parameters from a custom `/setEFSSMetadata` endpoint, thereby passing successful or error based responses to the wopiserver, which then allocated a new url to serve the etherpad user. 
```

Comment:
On top of what Andrei asked, please change `in-app` into `open file in app`, and then I suggest `targeted to enable users of a cloud storage opening and working with their files into Etherpad`.